.github/workflows/windows_unit_tests.yaml

```diff
@@ -53,7 +53,7 @@ jobs:
           . $env:USERPROFILE\Miniconda3\shell\condabin\conda-hook.ps1
           conda activate curr_py
           conda config --add channels conda-forge
-          conda install -q -y -c conda-forge python-graphviz graphviz
+          conda install -q -y -c conda-forge python-graphviz graphviz=9.0.0 pango=1.52.2
```

Comment:
If these are the only versions that work for these windows tests - what do you think of bumping graphviz and pango across the board since `graphviz==0.20.3` is the max on our latest dependencies check?

Reply:
Confusingly, (I think) there are 2 graphviz.

https://pypi.org/project/graphviz/#history, where the max version is 0.20.3. (conda-forge link: https://github.com/conda-forge/python-graphviz-feedstock/tree/main, max version is 0.20.3)

https://graphviz.org/download/source/, where the max is 11.0.0. (conda-forge link: https://github.com/conda-forge/graphviz-feedstock, max version is 9.0.0)

Reply:
Got it - I think it should be ok if it is consistent across all the windows (another spot would be [.github/workflows/windows_nightlies.yaml](https://github.com/alteryx/evalml/blob/7bd6ed693a0daa8662cd3a970f00dd663e9c8f4a/.github/workflows/windows_nightlies.yaml#L49)) and if it is mentioned specifically in [install.md](https://github.com/alteryx/evalml/blob/7bd6ed693a0daa8662cd3a970f00dd663e9c8f4a/docs/source/install.md?plain=1#L109) as well.